docs/Tor.md

```diff
@@ -16,7 +16,7 @@ brew install tor
 
 #### Windows:
   
-[Download the "Expert Bundle"](https://www.torproject.org/download/download.html) from Tor's website and extract it to `C:\tor`.
+[Download the "Expert Bundle"](https://www.torproject.org/download/tor/) from Tor's website and extract it to `C:\tor`.  A link to the "Expert Bundle" is on the "Download Tor Source Code" page.
```

Comment:
The following is sufficient: the link you provided gets us to a page where "Windows Expert Bundle" is very clearly visible.

```suggestion
[Download the "Expert Bundle"](https://www.torproject.org/download/tor/) from Tor's website and extract it to `C:\tor`.
```